static/src/stylesheets/module/email/_form.scss

```diff
@@ -300,6 +302,26 @@
     .email-sub__text-input {
         border-color: $guardian-brand;
     }
+
+}
+
+// PLAIN-DARK MODS
+.email-sub__form--plaindark {
+    padding: 0;
+
+    .email-sub__submit-input {
+        background-color: #DDDBD7;
```

Comment:
You can find the names of the colour variables here: http://guardian.github.io/pasteup-palette/demo/ - Should generally try and use the vars (especially for #333 below), and if we're in frontend try and keep the styles to the brand and generic (rather than a custom one for this one interactive)